endpoints/info/bidders.go

```diff
@@ -94,3 +138,32 @@ func prepareBiddersResponseEnabledOnly(bidders config.BidderInfos, aliases map[s
 
 	return json.Marshal(bidderNames)
 }
+
+func prepareBiddersResponseEnabledOnlyBaseOnly(bidders config.BidderInfos) ([]byte, error) {
```

Comment:
Does this function need the word Only in it twice? Would `prepareBiddersResponseEnabledBaseOnly` be better? Or is it better to clearly state that we're looking for only endpoint Enabled and only Base adapters?

Reply:
I don't think it matters too much. The names are intentional to match up with the `EnabledOnly` and `BaseOnly` flags.